happi/item.py

```diff
@@ -59,8 +59,7 @@ class EntryInfo:
     ------
```

Comment:
Is it time to remove `happi/device.py`?

Reply:
I think so, but I think we should do pcdsdevices first (it relies on `happi.Device` frustratingly)

Reply:
oops

Reply:
I'll give this a go

Reply:
Confirmed that branch works with this happi PR 👍